src/table/cells/handle-key-press.js

```diff
@@ -144,6 +150,7 @@ export const bodyHandleKeyPress = (
     }
     // Esc: Cancels selections
     case 'Escape': {
+      if (!isAnalysisMode || !selState.rows.length) break;
```

Comment:
do we really need this line?

Reply:
yes. if you press escape when there are no selections, the listener on the tableWrapper should catch the escape event and blur.

Reply:
I'll change the comment to reflect that

Reply:
done